helpers/render.py

```diff
@@ -284,6 +347,14 @@ def render_animation(args, anim_args, animation_prompts, root):
             
             # apply color matching
             if anim_args.color_coherence != 'None':
+                # video color matching
+                hybrid_video_available = anim_args.hybrid_video_composite or anim_args.hybrid_video_motion in ['Optical Flow', 'Affine', 'Perspective']
+                if anim_args.color_coherence == 'Video Input' and hybrid_video_available:
```

Comment:
wouldn't this line never be expected to catch? up at line 310 prev_sample needs to be set for this block to trigger, from what I understand prev_sample doesn't get set with video inits. UNLESS you're resuming animations which I interpreted to be a bug so I fixed it in this PR: https://github.com/deforum-art/deforum-stable-diffusion/pull/72

lmk what you think, and if I'm missing something with prev_sample

Reply:
In fact, in my local dev render_animation func, I have a line that sets color match sample to None if the strength is 0.  On the next cycle, this is the routine that picks up a new color match sample. 

As for prev_sample, well it's there every frame.

Oh, but maybe you mean prev_sample doesn't get set in Video Input mode...  I dunno.  My hybrid stuff works in 2D/3D mode, not Video Input mode.

Reply:
Yes okay cool that's what I thought might be getting confused for me, the 2D/3D mode is what you're using for hybrid so this is the section gets used, thanks for explaining!